integrations/sonarqube/main.py

```diff
@@ -94,10 +87,15 @@ async def handle_sonarqube_webhook(webhook_data: dict[str, Any]) -> None:
 
 @ocean.on_start()
 async def on_start() -> None:
+    if not ocean.integration_config.get("sonar_organization_id"):
+        if not is_onpremise_deployment():
+            raise ValueError(
+                "Organization ID is required for SonarCloud. Please specify a valid sonarOrganizationId in the configuration file"
```

Comment:
```suggestion
                "Organization ID is required for SonarCloud. Please specify a valid sonarOrganizationId"
```
Most times users don't run the integrations using the config.yml as we do locally, but rather through passing env / helm chart.